src/main/java/camp/nextstep/edu/rebellion/racing/domain/RacingCar.java

```diff
@@ -9,9 +9,14 @@
     private final String name;
     private int position;
 
-    public RacingCar(String name){
+    private RacingCar(String name, int position) {
         validateName(name);
         this.name = name;
+        this.position = position;
+    }
+
+    public RacingCar(String name){
+        this(name, 0);
```

Comment:
사소하진 0이라는 숫자가 코드 안에서 읽힌다면 무엇을 뜻하는지 그리고 왜? 라는 의문을 가지기 쉬운 것 같아요!
매직 넘버를 좀 더 의미있게, 그리고 관리하기 쉽도록 상수로 치환하면 좋을 것 같아요 :)
```suggestion
        this(name, DEFAULT_DISTANCE);
```